server/models/notifications.js

```diff
@@ -0,0 +1,54 @@
+const getNotificationsModel = (sequelize, DataTypes) => {
+  const flags = {
+    freezeTableName: true
+  };
+
+  const NotificationShema = {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      allowNull: false,
+      defaultValue: DataTypes.UUIDV4,
+      unique: true
+    },
+    userId: {
+      type: DataTypes.UUID
+    },
+    message: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    isSeen: {
+      type: DataTypes.BOOLEAN,
+      defaultValue: false,
+      allowNull: false
```

Comment:
No need for the `allowNull: false` since you have given the field a `defaultValue`. Please remove.

Reply:
Allow null ensures that the value can never be null because If a null value is specified, it will override the false/defaultValue